configure

```diff
@@ -49,6 +49,76 @@ while true; do
   # Retry
 done
 
+## Set up MKL related environment settings
+
+while [ "$TF_NEED_MKL" == "" ]; do
+	fromuser=""
+  read -p "Do you wish to build TensorFlow with MKL support? [y/N] " INPUT
```

Comment:
Will this poll the user by default if they want to use MKL support?  I had thought that we wanted to leave this disabled until everything was submitted and verified to be working fine.